I do magic/promises-callbacks-js/src/js/mustache.js

```diff
@@ -0,0 +1,16 @@
+// Mustache template for the agents
+var TEMPLATE = '{{#companies}}\
```

Comment:
Shouldn't be this a const? Also, all caps labels should be const, not vars.

Reply:
And this should be a mustache template file, not inlined in the JS. Remember not to mix languages, is not a good practice. Ask Kristal about how she used an external mustache file.

Reply:
Done